Batch.py

```diff
@@ -104,28 +113,28 @@ def backward(self, optimizer):
         optimizer.step()
 
     def display(self):
-        if args.display:
+        if ARGS.display:
             o = self.outputs[0].data.cpu().numpy()
             t = self.target_data[0].cpu().numpy()
 
-            print('Loss:', dp(self.loss.data.cpu().numpy()[0], 5))
+            print('Loss:', np.round(self.loss.data.cpu().numpy()[0], decimals=5))
             a = self.camera_data[0][:].cpu().numpy()
             b = a.transpose(1, 2, 0)
-            h = shape(a)[1]
-            w = shape(a)[2]
-            c = zeros((10 + h * 2, 10 + 2 * w, 3))
+            h = np.shape(a)[1]
+            w = np.shape(a)[2]
+            c = np.zeros((10 + h * 2, 10 + 2 * w, 3))
             c[:h, :w, :] = z2o(b[:, :, 3:6])
             c[:h, -w:, :] = z2o(b[:, :, :3])
             c[-h:, :w, :] = z2o(b[:, :, 9:12])
             c[-h:, -w:, :] = z2o(b[:, :, 6:9])
             mi(c, 'cameras')
             print(a.min(), a.max())
-            figure('steer')
-            clf()
-            ylim(-0.05, 1.05)
-            xlim(0, len(t))
-            plot([-1, 60], [0.49, 0.49], 'k')
-            plot(o, 'og')
-            plot(t, 'or')
+            plt.figure('steer')
+            plt.clf()
+            plt.ylim(-0.05, 1.05)
+            plt.xlim(0, len(t))
+            plt.plot([-1, 60], [0.49, 0.49], 'k')
+            plt.plot(o, 'og')
+            plt.plot(t, 'or')
```

Comment:
I already added a comment in there  to clarify for those new to mpl, the argument is just format string or something so idk if it will help very much.